ipld/amt/src/lib.rs

```diff
@@ -8,7 +8,7 @@
 
 mod amt;
 mod bitmap;
-mod block_store;
+pub mod block_store;
```

Comment:
BlockStore is already being exported? Why export the module here?

Reply:
As mentioned, there are compile errors when trying to use the BlockStore trait without the module being public.

Reply:
Nah you don't need this just refer to it as `amt::BlockStore` and not `amt::blockstore::BlockStore`